Languages/IronPython/IronPython/Runtime/PythonFile.cs

```diff
@@ -1208,7 +1208,7 @@ public PythonFile(CodeContext/*!*/ context)
 
             // remember the original mode for error reporting
             string inMode = mode;
-
+            
```

Comment:
I have nothing against add/removing empty lines to increase readability, but adding spaces at random places is in general not a good idea. There is a plug-in or settings for this in VS. It is also quite prominent when you look at commits in git gui.